api/api.md

```diff
@@ -2,9 +2,10 @@
 
 # Adobe I/O Events API
 
-For at a glance view of our API, please refer to our [Adobe I/O Events `Open API` documentation ](https://www.adobe.io/apis/experienceplatform/events/ioeventsapi.html#!adobedocs/adobeio-events/master/events-api-reference.yaml).
+For at a glance view of our API, please refer to our [Adobe I/O Events `OpenAPI` documentation ](https://www.adobe.io/apis/experienceplatform/events/ioeventsapi.html#!adobedocs/adobeio-events/master/events-api-reference.yaml).
 
 We also provide additional documentation on:
 * our [Journaling API](journaling_api.md)
-* our [Provider API](provider_api.md)
-* our [Registration API](registration_api.md)
\ No newline at end of file
+* our [Registration API](registration_api.md)
+* our [Management API](provider_api.md)
```

Comment:
The title of the page is wrong, this page talks about Providers API, so it shouldn't be named Management API

Reply:
done https://github.com/AdobeDocs/adobeio-events/pull/77/commits/067dbf5c96179cf4f2b7272b0349889c5b8b1b10